pom.xml

```diff
@@ -147,6 +147,13 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.0</version>
```

Comment:
Could you move the version numbers into the `properties` section? It's easier to manage the dependencies this way.